src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/AttributeMapper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright 2010 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting;
+
+use Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Dto\AttributeListDto;
+use Surfnet\StepupSelfService\SelfServiceBundle\Service\RemoteVetting\Value\Attribute;
+
+class AttributeMapper
+{
+    /**
+     * @var IdentityProviderFactory
+     */
+    private $identityProviderFactory;
+
+    public function __construct(IdentityProviderFactory $identityProviderFactory)
+    {
+        $this->identityProviderFactory = $identityProviderFactory;
+    }
+
+    /**
+     * @param string $identityProviderName
+     * @param AttributeListDto $localAttributes
+     * @param AttributeListDto $externalAttributes
+     * @return AttributeListDto
+     */
+    public function map($identityProviderName, AttributeListDto $localAttributes, AttributeListDto $externalAttributes)
+    {
+        $attributeMapping = $this->identityProviderFactory->getAttributes($identityProviderName);
+
+        $mappedAttributes = [];
+        $attributesToMap = [];
+
+        /** @var Attribute $attribute */
```

Comment:
This doc can be removed if correct @return annotation is used on the `getAttributes` method right?

Reply:
Done